photo-helper/src/main/java/io/intrepid/photohelper/ContextDelegate.java

```diff
@@ -0,0 +1,17 @@
+package io.intrepid.photohelper;
+
+import android.content.ContextWrapper;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+
+/**
+ * Created by GuyIntrepid on 6/1/17.
+ */
+
+interface ContextDelegate {
+    void requestPermissions(@NonNull String[] permissions, int requestCode);
+
+    ContextWrapper getContext();
```

Comment:
I must have made a stupid mistake somewhere - I was convinced we were using some method that only existed in ContextWrapper.  Must have been looking at the wrong file.